### PR TITLE
ARUHA-1398 reduce logging when user committed with unknown stream ID

### PR DESCRIPTION
**Pull request by lmontrieux**
ARUHA-1398: Reduce logging when user commits with unknown stream ID

## Description
HOTFIX - when user commits with unknown stream ID, only log exception message

## Review
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/controller/CursorsController.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ public CursorsController(final CursorsService cursorsService,
     @ExceptionHandler(InvalidStreamIdException.class)
     public ResponseEntity<Problem> handleInvalidStreamId(final InvalidStreamIdException ex,
                                                          final NativeWebRequest request) {
-        LOG.warn("Stream id {} is not found", ex.getStreamId(), ex);
+        LOG.warn("Stream id {} is not found", ex.getStreamId(), ex.getMessage());
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/controller/CursorsController.java]: @lmontrieux 
Wow... Strange. I think in this manner you need to add addional `{}` to log message.

Comment (lmontrieux) [reply src/main/java/org/zalando/nakadi/controller/CursorsController.java]: 088ddfd

**Comment by antban** [comment]
плюс один

**Comment by antban** [comment]
:+1:

**Comment by lmontrieux** [comment]
plus un

**Comment by lmontrieux** [comment]
👍

**Pull request merged**